source/structured-streaming.txt

```diff
@@ -307,6 +428,50 @@ Stream to MongoDB from a CSV File
 
    tabs:
 
+     - id: java-sync
+       content: |
+
+         To create a :ref:`write stream <write-structured-stream>` to 
+         MongoDB from a ``.csv`` file, first create a `DataStreamReader <https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/streaming/DataStreamReader.html>`__ 
```

Comment:
Issue: these instructions refer to the DataStreamReader and DataStreamWriter rather than a "readstream property" or "writestream property" as used in previous descriptions in this PR.

Suggestion: make them uniform.

Reply:
Updating Read and Write config sections to use DataStreamReader and Writer